ipvs/ipvs.go

```diff
@@ -82,6 +86,14 @@ func New(path string) (*Handle, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Add operation timeout to avoid deadlocks
+	tv := syscall.NsecToTimeval(netlinkSocketsTimeout.Nanoseconds())
+	if err := sock.SetReceiveTimeout(&tv); err != nil {
+		return nil, err
+	}
+	if err := sock.SetReceiveTimeout(&tv); err != nil {
```

Comment:
why is this done twice ?

Reply:
good catch it should be ReceiveTimeout and SendTimeout

Reply:
Shouldn't `netlinkSendSocketTimeout` be `netlinkRecvSocketsTimeout` here ?